sdk/maps/Azure.ResourceManager.Maps/src/autorest.md

```diff
@@ -29,6 +28,9 @@ prepend-rp-prefix:
   - Creator
   - CreatorProperties
   - KeyType
+  - LinkedResource
+  - SigningKey
+  - CorsRule
```

Comment:
The original path is `CorsRules` not `CorsRule`, this is not taking effect.
Please add below to `rename-mapping` section instead:
```yaml
CorsRules: MapsCorsRule
```

Reply:
Updated, but I'm a little confused I expect this is to change the class `public partial class CorsRule` to `public partial class MapsCorsRule`, no?

Reply:
The JSON path is [CorsRules](https://github.com/Azure/azure-rest-api-specs/blob/6b08774c89877269e73e11ac3ecbd1bd4e14f5a0/specification/maps/resource-manager/Microsoft.Maps/preview/2021-12-01-preview/maps-management.json#L1394C9-L1394C9), generator changed the naming to `CorsRule` during the generation. That's why CorsRule does not work since it doesn't match with the JSON path in swagger.

Reply:
So I guess I should add the rule to `prepend-rp-prefix`, right?

```yaml
prepend-rp-prefix:
  - CorsRule
```

Reply:
Yes, this works as well.